README.md

```diff
@@ -50,16 +51,17 @@ This will produce files inside `lib/generated` directory.
 
 #### Upload main ARB file
 
-      flutter pub run intl_utils:localizely_upload_main --project-id <PROJECT_ID> --api-token <API_TOKEN>
+      flutter pub run intl_utils:localizely_upload_main --project-id <PROJECT_ID> --api-token <API_TOKEN> [--arb_dir <ARB_DIR>]
```

Comment:
Let's use dashes in arg names, just to be consistent with the rest of the code.

`--arb_dir <ARB_DIR>` -> `--arb-dir <ARB_DIR>`